source/MapPlanetCard.cpp

```diff
@@ -203,13 +225,20 @@ const Planet *MapPlanetCard::GetPlanet() const
 
 
 
+void MapPlanetCard::Select(bool select)
+{
+	isSelected = select;
+}
+
+
+
 void MapPlanetCard::Highlight(double availableSpace) const
 {
 	const Interface *planetCardInterface = GameData::Interfaces().Get("map planet card");
 	const double width = planetCardInterface->GetValue("width");
 
 	FillShader::Fill(Point(Screen::Left() + width / 2., yCoordinate + availableSpace / 2.),
-		Point(width, availableSpace), *GameData::Colors().Get("faint"));
+		Point(width, availableSpace), Color(0.f, .3f));
```

Comment:
If you need a new color then you should define it in the interface file instead of hardcoding it.

Reply:
It's already done this way in mapsalespanel.cpp, line 367, but indeed it shouldn't be, I'll make a color for both